packages/components/README.md

```diff
@@ -7,48 +7,217 @@
+They are distributed along with the components code in the `dist` folder of the library.
+
+```js
+import '@plone/components/basiq.css';
+```
+
+or selectively:
+
+```js
+import '@plone/components/src/styles/basiq/TextField.css';
+```
+
+## Theming
+
+The idea is that you can use the BasiQ styles as baseline while building the theme of your site.
+You can take advantage of them as they are very thin and basic (almost vanilla CSS for the components).
+Using them as a baseline will allow you to quickly build your theme around them.
+BasiQ provides a simple, yet powerful set of tokenized custom CSS properties that will help you customize your own styles on the top of BasiQ.
+You can override them in your classes while maintaining them for others.
+
+### Quanta
+
+This package also features the Quanta components.
+The Quanta theme is an example of it.
+These components use BasiQ as baseline (not only in styling, but also in the component side), so they are built upon it and reusing the CSS and custom CSS properties in there.
+Quanta is built upon the basic styles of BasiQ in an additive way, so the use of the Quanta CSS implies to use it upon BasiQ styling.
+You could take Quanta as example to build your own layer of styles over BasiQ for your theme.
+
+In order to use a theme built upon BasiQ, you need to import both BasiQ and the theme CSS, in this order:
+
+```js
+import '@plone/components/basiq.css';
+import '@plone/components/quanta.css';
+```
+
+You have the option of doing it selectively per component too:
+
+```js
+import '@plone/components/src/styles/basiq/TextField.css';
+import '@plone/components/src/styles/quanta/TextField.css';
+```
+
+Take a look at the implementation of the Quanta components, using the BasiQ ones as baseline in the `quanta` folders.
+
+Alternatively, as RAC allows, you can also drop your own basic set of styles.
+You can take the BasiQ styles as reference.
+You can even bring your own CSS framework and make BasiQ get the styling in there using the utilities of your choice.
+It's even possible to use TailwindCSS for styling the components in this package, using the RAC styling approach.
 
 ## Components list
 
-### Fundamentals
+### BasiQ
 
+- Button
+- Checkbox
+- CheckboxField
+- CheckboxGroup
+- Combobox
 - Container
-- Input
+- Dialog
+- Form
+- GridList
 - Icon
-
-### Structural
+- Link
+- Listbox
+- Menu
+- Modal
+- NumberField
+- Popover
+- RadioGroup
+- Select
+- Switch
+- Tabs
+- TagGroup
+- TextField
+- TextAreaField
+- ToggleButton
+- Toolbar
+- BlockToolbar
+- Tooltip
+
+### Viewlets
 
 - Breadcrumbs
 
+### Views
+
+- DefaultBlockView
+- RenderBlocks
+
+### Quanta
+
+- TextField
+- TextAreaField
+
+## Quanta icons
+
+This package provides an implementation of the Quanta Icon set in React components.
+They can be used directly in your components as:
+
+```tsx
+import { ChevronupIcon } from '../Icons/ChevronupIcon';
+
+const MyComponent = (props) => (
+  <Button>
```

Comment:
I added `aria-label` on the icons. Question is if we want them to provide one by default or not. Is that what you meant?

Reply:
I have to admit I did not notice the `aria-label` on the icons, sorry. I am not entirely sure it's enough though, for these reasons:

- `title` also provides a tooltip which is nicer for visual users that only see an icon instead (even if totally understandable like a chevron in this case)
- iirc the title or label should be on the button, because screen readers will read that as the button label and it is more straightforward to understand for the user. Also, the icon itself usually has `aria-hidden="true"` so I'm not sure if the label would be read, I haven't tested it.

To answer you question, I only meant to have a good example in the docs, I did not mean to change the components code.

Reply:
OK! fortunately RAC will help with all these, since they already cover a lot, but these guidelines will be great to have sumarized somewhere (probably in the components docs too).

Reply:
Btw, `title` is not in the RAC Button props... that is indicator that having the title is no longer the best practice or the browsers have now support for aria instead...

Reply:
https://react-spectrum.adobe.com/react-spectrum/Button.html#accessibility

Reply:
Also on that regard, I plan to make a test on A11y with AXE for all components.

Reply:
> Btw, `title` is not in the RAC Button props... that is indicator that having the title is no longer the best practice or the browsers have now support for aria instead...

Yes, this is correct. Also, `aria-label` always takes precedence when set, it's the best way to go. What I kind of dislike of this new spec is that `title` also added the tooltip and I really don't get why that is not an important feature for people making decisions on these topics :D anyway, I'm happy if we stick to `aria-label` here